### PR TITLE
Add Sharp Tune

### PR DESCRIPTION
**Pull request by MD-AZMAL**
Add my music player to apps

### CODE REVIEW

File: apps/sharp-tune/sharp-tune.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Sharp-Tune
+description: 'Lightweight Cross-platform music player build upon the node using the electron framework.'
```
Comment (zeke) [inline apps/sharp-tune/sharp-tune.yml]: No need to mention node and electron in the description. How about just `Lightweight cross-platform music player`?

Comment (MD-AZMAL) [reply apps/sharp-tune/sharp-tune.yml]: sounds great!

Comment (MD-AZMAL) [reply apps/sharp-tune/sharp-tune.yml]: Please check i have made the necessary changes .......... thanks for the advice

Comment (MD-AZMAL) [reply apps/sharp-tune/sharp-tune.yml]: @zeke i have updated the screenshot for the app as well and added a few more keywords have a look!

**Comment by zeke** [comment]
Thanks!

**Pull request merged**